utils/crates-io/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+//! crates-io-manager library
+#![deny(missing_docs)]
+#![deny(unused_crate_dependencies)]
+
+mod manifest;
+mod publisher;
+pub mod rename;
+mod version;
+
+pub use self::{manifest::ManifestWithPath, publisher::Publisher, version::verify};
+use anyhow::Result;
+use std::process::{Command, ExitStatus};
+
+/// Required Packages without local dependencies.
+pub const SAFE_DEPENDENCIES: [&str; 10] = [
+    "actor-system-error",
+    "galloc",
+    "gear-stack-buffer",
+    "gear-core-errors",
+    "gear-common-codegen",
+    "gear-wasm-instrument",
+    "gmeta-codegen",
+    "gsdk-codegen",
+    "gstd-codegen",
+    "gsys",
+];
+
+/// Required packages with local dependencies.
+pub const STACKED_DEPENDENCIES: [&str; 5] =
+    ["gcore", "gmeta", "gear-core", "gear-utils", "gear-common"];
+
+/// Packages need to be published.
+pub const PACKAGES: [&str; 5] = ["gear-wasm-builder", "gstd", "gsdk", "gclient", "gcli"];
```

Comment:
gtest?

Reply:
> gtest could be more complex than the two packages in this PR, since this PR already includes a refactor of the manager, #3501 should be in another PR

waiting for that 👍🏻